src/BuildScriptGenerator/DefaultDockerfileGenerator.cs

```diff
@@ -103,5 +103,15 @@ private string GenerateRuntimeTag(string version)
 
             return $"{split[0]}.{split[1]}";
         }
+
+        private string ConvertToRuntimeName(string platformName)
```

Comment:
im bit confused on the reason of having this function instead of having a key value pair  for dotnetcore, we can discuss offline as well

Reply:
This part is tricky because the name of the platform in our project is called `dotnet`, but the runtime image is called `dotnetcore`, so we need to add in a small layer to transform the name. I'm not sure there's an easy way for us to change the platform name to `dotnetcore` without breaking existing users in some way, but if there's a way to do this, or if you have idea of how we should resolve this issue, let me know.